### ISSUE TITLE
Use of app password  for nextcloud

### ISSUE THREAD

**Issue by ernesst** [yunit-io/account-plugins#1]
I think the bug should be present also in Yunit, didn't verify.
according to https://github.com/nextcloud/server/issues/6626

I've submitted a bug fix to Ubports project : 
https://github.com/ubports/account-plugins/issues/1

and the pull for Ubports : https://github.com/ubports/account-plugins/pull/2

I don't know how to submit my pull here, should i also fork your repository ?

**Comment by jsalatas** [comment]
> I don't know how to submit my pull here, should i also fork your repository ?

Yes, you may want to submit it here as well, after forking our repository :\

I guess we need to resolve such issues with ubports :\
